python/mxnet/image/image.py

```diff
@@ -585,14 +586,12 @@ def random_size_crop(src, size, area, ratio, interp=2, **kwargs):
         area = (area, 1.0)
     for _ in range(10):
         target_area = random.uniform(area[0], area[1]) * src_area
-        new_ratio = random.uniform(*ratio)
+        log_ratio = (math.log(ratio[0]), math.log(ratio[1]))
```

Comment:
you don't need to import `math`, use `np` which is already imported

Reply:
done!